lib/error_message/serializer.ex

```diff
@@ -77,13 +77,38 @@ defmodule ErrorMessage.Serializer do
   end
 
   defp ensure_json_serializable(details) when is_tuple(details) do
-    details |> Tuple.to_list |> ensure_json_serializable
+    details |> Tuple.to_list() |> ensure_json_serializable
+  end
+
+  defp ensure_json_serializable(fun) when is_function(fun) do
+    case :erlang.fun_info(fun, :module) do
+      {:module, :erl_eval} ->
+        build_function_metadata(:erl_eval, "anonymous function", fun)
+
+      {:module, module} ->
+        {:name, name} = :erlang.fun_info(fun, :name)
+
+        module
+        |> Kernel.to_string()
+        |> String.replace("Elixir.", "")
```

Comment:
Is it worth doing `is_atom` in one of the cases to cover erlang modules that don't need a Elixir. prefix

Reply:
@MikaAK I pushed a new code version which simplifies serialisation.